data_layer/celestia/celestia.go

```diff
@@ -30,6 +30,11 @@ type Celestia struct {
 	rpcEndpoint string
 }
 
+func (c2 *Celestia) GetStatus(c config.RollappConfig) string {
```

Comment:
so what gonna be the status of Celestia for now?
where is the balance check?

Reply:
Will be implemented when their new version will be available and working